src/system/config.ts

```diff
@@ -103,6 +104,27 @@ const COSMERE: CosmereRPGConfig = {
         },
     },
 
+    injuries: {
+        [InjuryDuration.FleshWound]: {
+            label: 'Flesh Wound',
+            durationFormula: '1',
+        },
+        [InjuryDuration.ShallowInjury]: {
+            label: 'Shallow Injury',
+            durationFormula: '1d6',
+        },
+        [InjuryDuration.ViciousInjury]: {
+            label: 'Vicious Injury',
+            durationFormula: '6d6',
+        },
+        [InjuryDuration.PermanentInjury]: {
+            label: 'Permanent Injury',
+        },
+        [InjuryDuration.Death]: {
+            label: 'Death',
+        },
+    },
```

Comment:
These labels need to be localized

Reply:
Whoops! Didn't notice that localization is in because I'm forked from main. I'll get those changes in shortly